### DISCUSSION TITLE
suddenly stopped working and strange report

### DISCUSSION THREAD

**Discussion by avispiel** [openMVG/openMVG#2081] [Q&A] [answered]
Hi.
I had the program working and then I recompiled and it doesnt work.
strange this I notices: the report (and also sfm_data.bin) has 52 views but only 51 poses.
when I iterate through views and check each pose id, one of them referes to a non existing pose
[reconstruction_global.zip](https://github.com/openMVG/openMVG/files/8908524/reconstruction_global.zip)
.

**Comment by pmoulon** [comment, marked as answer]
Seems like one camera pose was not found at the SFM stage.
Not: this is normal to have all the view listed but only valid poses kept. This way it is easy for users to see how many images are used in the project and how many were positioned in 3d.

Else regarding was working before and not now.. not sure if something changed on your machine. Did you compile OpenMVG from scratch?

**Reply by pmoulon** [reply]
Marked as answered due to lack of feedback, feel free to recomment if needed.
